etc/mqttwarn.openrc

```diff
@@ -0,0 +1,14 @@
+#!/sbin/openrc-run
```

Comment:
what is this whole file for?
if it is required, should this be in contrib/ ?

Reply:
This file is the init script for openrc based systems like Alpine Linux

Reply:
I think it should be in same directory as mqttwarn.service